libs/api/libbpf_map.cpp

```diff
@@ -189,3 +189,9 @@ bpf_map_update_elem(int fd, const void* key, const void* value, uint64_t flags)
 {
     return libbpf_err(ebpf_map_update_element(fd, key, value, flags));
 }
+
+int
+bpf_map_lookup_elem(int fd, const void* key, void* value)
```

Comment:
Need to add this API to ebpfapi\Source.def (looks like bpf_map_update_elem is missing from there too)